addons/im_livechat/views/res_users_views.xml

```diff
@@ -9,9 +9,15 @@
             <field name="inherit_id" ref="base.view_users_form_simple_modif"/>
             <field name="arch" type="xml">
                 <xpath expr="//field[@name='tz']" position="after">
+                    <field name="has_access_livechat" invisible="1"/>
                     <field name="livechat_username" string="Online Chat Name"
                         readonly="0"
-                        invisible="share"/>
+                        invisible="share or not has_access_livechat"/>
+                    <field name="livechat_lang_ids" string="Online Chat Language"
+                        readonly="0"
```

Comment:
Isn't `readonly="0"` the default, especially if `readonly=False` is set in python?